keyboards/ergodox/infinity/infinity.h

```diff
@@ -0,0 +1,110 @@
+#ifndef KEYBOARDS_INFINITY_ERGODOX_INFINITY_ERGODOX_H_
```

Comment:
Yes, I will change that a bit later today. The KEYBOARDS_ERGODOX_INFINITY_PART is not exactly according to the documentation either, but Eclipse, which I'm using adds the folder names so I have left them. 

This mixup was probably because I moved stuff around.